backend/src/serverless/integrations/workers/twitterWorker.ts

```diff
@@ -78,6 +80,8 @@ async function twitterWorker(body: TwitterIntegrationMessage): Promise<TwitterOu
 
     const followersSet: Set<string> = new Set(integrationUpdated.settings.followers)
 
+    await new MemberAttributeSettingsService(userContext).createPredefined(TwitterMemberAttributes)
```

Comment:
If I understand this correctly - every time a worker is being run we again try to create attribute settings in the database?
Doesn't seem like the best way to do it - perhaps it would be better if we do this just once when integration is created or something like that and then prepare a way to do it on demand to run it when we manually change the predefined attributes? It seems like a lot of processing for something that will be actually done in less than 1% of the runs.

Reply:
Now we're checking integration.settings.updateMemberAttributes boolean to decide update or not, after this function updateMemberAttributes is set to false again